drv/Iterator.drv

```diff
@@ -58,6 +58,7 @@ public interface KEY_ITERATOR KEY_GENERIC extends Iterator<KEY_GENERIC_CLASS> {
 	@Deprecated
 	@Override
 	default KEY_CLASS next() {
+		assert false : "Use the primitive next..() method instead.";
```

Comment:
Can we put NEXT_KEY here?

Reply:
I already tried that, just like the c preprocessor, it doesn't do substitutions inside string